includes/settings.php

```diff
@@ -5,13 +5,33 @@
  * 
  * Note       : Please do not commit your credentials
  */
+
+define("HOST", "http://localhost/FirstAide-web/");
+
+$_settings = array();
+
 $_settings['reCaptcha'] = array();
-$_settings['reCaptcha']['client_key'] = '';
-$_settings['reCaptcha']['server_key'] = '';
+$_settings['reCaptcha']['client_key'] = getenv('client_key') ? getenv('client_key') : '';
+$_settings['reCaptcha']['server_key'] = getenv('server_key') ? getenv('server_key') : '';
 
 $_settings['twilio'] = array();
-$_settings['twilio']['account_sid'] = '';
-$_settings['twilio']['auth_token'] = '';
-$_settings['twilio']['number'] = '';
+$_settings['twilio']['account_sid'] = getenv('account_sid') ?  getenv('account_sid') : '';
+$_settings['twilio']['auth_token'] = getenv('auth_token') ? getenv('auth_token') : '';
+$_settings['twilio']['number'] = getenv('number') ?  getenv('number') : '';
+
+//'heroku' for production, 'debug' for running locally
+$_settings['deploy_mode'] = strpos($_SERVER['REQUEST_URI'], 'herokuapp.com') ? 'heroku' : 'debug';
 
+if ($_settings['deploy_mode'] == 'heroku') {
```

Comment:
You shouldn't tie the database down to just one deployment target. Use something like
```
if ($development) {
    //Set the DB details
} else {
    //Fetch the DB details from env variable
}
```

Or even better you can get rid of the if block and always load the configuration from the env file.